packages/manager/apps/dedicated/client/app/account/user/users/users.html

```diff
@@ -33,6 +37,7 @@
         data-ng-click="setAction('users/add/user-users-add', user)"
         data-ng-disabled="$ctrl.usersLoading"
         data-ng-class="{'disabled': $ctrl.usersLoading}"
+        data-ng-show="$ctrl.identityProvider == null"
```

Comment:
```suggestion
        data-ng-if="$ctrl.identityProvider == null"
```
This will avoid generating the element if it should not be displayed